etcd/etcd-live-cluster-reconfiguration.md

```diff
@@ -1,236 +1,294 @@
-member 1609b5a3a078c227 is unreachable: [http://10.0.1.2:2379] are all unreachable
-member 60e8a32b09dc91f1 is healthy: got healthy result from http://10.0.1.3:2379
+member fe2f75dd51fa5ff is healthy: got healthy result from http://10.240.0.1:2379
+failed to check the health of member 1609b5a3a078c227 on http://10.240.0.2:2379: Get http://10.240.0.2:2379/health: dial tcp 10.240.0.2:2379: connection refused
+member 1609b5a3a078c227 is unreachable: [http://10.240.0.2:2379] are all unreachable
+member 60e8a32b09dc91f1 is healthy: got healthy result from http://10.240.0.3:2379
 cluster is healthy
 ```
 
-Run `etcdctl` from a working node, or use the [`ETCDCTL_ENDPOINT`][etcdctl-endpoint] environment variable or command line option to point `etcdctl` at any healthy member node.
 
-[Remove the failed member][etcdctl-member-remove] `10.0.1.2` from the etcd cluster. The remove subcommand informs all other cluster nodes that a human has determined this node is dead and not available for connections:
+1. [Remove the failed member][etcdctl-member-remove] `1609b5a3a078c227` from the etcd cluster.
 
-```sh
-$ etcdctl member remove 1609b5a3a078c227
-Removed member 1609b5a3a078c227 from cluster
-```
+    ```sh
+    $ etcdctl member remove 1609b5a3a078c227
+    Removed member 1609b5a3a078c227 from cluster
+    ```
+    The remove subcommand informs all other cluster nodes that a human has determined this node is dead and not available for connections.
 
-Then, on the failed node (`10.0.1.2`), stop the etcd2 service:
+2. Stop the etcd-member service on the failed node (`0.0.0.2`):
 
-```sh
-$ sudo systemctl stop etcd2
-```
+    ```sh
+    $ sudo systemctl stop etcd-member.service
+    ```
 
-Clean up the `/var/lib/etcd2` directory:
+3. Reinitialize the failed member.
 
-```sh
-$ sudo rm -rf /var/lib/etcd2/*
-```
+      ```sh
+        $ etcdctl member add node2 http://0.0.0.2:2380
+        Added member named node2 with ID 4fb77509779cac99 to cluster
 
-Check that the `/var/lib/etcd2/` directory exists and is empty. If you removed this directory accidentally, you can recreate it with the proper modes by using:
+        ETCD_NAME="node2"
+        ETCD_INITIAL_CLUSTER="demo-etcd-1=http://10.240.0.1:2380,demo-etcd-2=http://10.240.0.2:2380,node3=http://10.240.0.3:2380"
+        ETCD_INITIAL_CLUSTER_STATE="existing"
+        ```
 
-```sh
-$ sudo systemd-tmpfiles --create /usr/lib64/tmpfiles.d/etcd2.conf
-```
+4. Modify the existing systemd drop-in, `/etc/systemd/system/etcd-member.service.d/20-clct-etcd-member.conf` by replacing the node data with the appropriate information from the output of the `etcdctl member add` command executed in the last step.
 
-Next, reinitialize the failed member. Note that `10.0.1.2` is an example IP address. Use the IP address corresponding to your failed node:
+    ```yaml container-linux-config
+    etcd:
+      name: demo-etcd-1
```

Comment:
name: demo-etcd-2

to match the node being replaced via etcdctl member add/remove